### PR TITLE
Update simple media app doc 

### PR DESCRIPTION
**Pull request by kul3r4**
#### WHAT
Currently the sample from the doc is outdated and does not compile

#### WHY
Improve docs

#### HOW
Fix the code and make it compiling

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: docs/simple-media-app-guide.md [hunk 1]
```diff
@@ -121,23 +116,41 @@ init {
...
+        buttons = { }
 )
 ```
+Add `createVolumeViewModel` function to create a VolumeViewModel:
+
+```kotlin
+fun createVolumeViewModel(): VolumeViewModel {
+        val audioRepository = SystemAudioRepository.fromContext(application)
+        val vibrator: Vibrator = application.getSystemService(Vibrator::class.java)
+        //TODO Create a simpler volumeViewModel with just a context
```
Comment (luizgrp) [inline docs/simple-media-app-guide.md:148]: nit: do you want to merge this comment?

File: docs/simple-media-app-guide.md [hunk 2]
```diff
@@ -121,23 +116,41 @@ init {
...
-    },
-    controlButtons = { playerUiState ->
-        PodcastControlButtons(
-            playerViewModel = viewModel,
-            playerUiState = playerUiState,
-        )
-    },
-    buttons = { }
+        playerViewModel = viewModel,
+        volumeViewModel = createVolumeViewModel(),
```
Comment (luizgrp) [inline docs/simple-media-app-guide.md:128]: nit: I'd suggest to create a variable for the volumeViewModel and add it under the comment on line 123 that says that ViewModels should not be created like this

**Review by luizgrp — APPROVED** [review]
Thanks!

File: docs/simple-media-app-guide.md (hunk 2 above)
Comment (kul3r4) [reply docs/simple-media-app-guide.md]: good point, done

File: docs/simple-media-app-guide.md (hunk 1 above)
Comment (kul3r4) [reply docs/simple-media-app-guide.md]: removed, thanks

**Pull request merged**